package.json

```diff
@@ -73,7 +73,8 @@
     "react-motion": "0.5.2",
     "react-required-if": "^1.0.3",
     "react-resize-aware": "2.7.0",
-    "react-router": "3.2.0"
+    "react-router": "3.2.0",
+    "terser-webpack-plugin": "^4.2.3"
```

Comment:
I guess this could be under `devDependencies` as well - just like `uglifyjs-webpack-plugin` before?

Reply:
Agreed